worker/categorize.py

```diff
@@ -395,7 +395,7 @@ def _is_viable_candidate(self, phrase):
         words = phrase.split()
 
         # Ignore stop words
-        if words[0].lower() in STOP_WORDS:
+        if len(words) == 0 or words[0].lower() in STOP_WORDS:
```

Comment:
I would put a check for None/empty at the beginning of the function that returns false instead.